.gitmodules

```diff
@@ -290,3 +290,6 @@
 [submodule "contrib/morton-nd"]
 	path = contrib/morton-nd
 	url = https://github.com/morton-nd/morton-nd
+[submodule "contrib/xxHash"]
```

Comment:
Just in case - isn't it included in zstd?

Reply:
it is, but in the same way as lz4: zstd contains a snapshot of these files of some undetermined version in the past. I think it is better to use the actual lib